consensus/wal_generator.go

```diff
@@ -1,229 +0,0 @@
-package consensus
-
-import (
-	"bufio"
-	"bytes"
-	"fmt"
-	"io"
-	mrand "math/rand"
-	"path/filepath"
-	"testing"
-	"time"
-
-	"github.com/lazyledger/lazyledger-core/abci/example/kvstore"
-	cfg "github.com/lazyledger/lazyledger-core/config"
-	"github.com/lazyledger/lazyledger-core/libs/db/memdb"
-	"github.com/lazyledger/lazyledger-core/libs/log"
-	"github.com/lazyledger/lazyledger-core/privval"
-	"github.com/lazyledger/lazyledger-core/proxy"
-	sm "github.com/lazyledger/lazyledger-core/state"
-	"github.com/lazyledger/lazyledger-core/store"
-	"github.com/lazyledger/lazyledger-core/types"
-)
-
-// WALGenerateNBlocks generates a consensus WAL. It does this by spinning up a
-// stripped down version of node (proxy app, event bus, consensus state) with a
-// persistent kvstore application and special consensus wal instance
-// (byteBufferWAL) and waits until numBlocks are created.
-// If the node fails to produce given numBlocks, it returns an error.
-func WALGenerateNBlocks(t *testing.T, wr io.Writer, numBlocks int) (err error) {
```

Comment:
These are only used in consensus/wal_test.go. Hence, I've moved them there (and it also helped with re-using the same ipfsAPI object in TestMain, see consensus/replay_test.go).